core/domain/suggestion_registry.py

```diff
@@ -503,42 +503,42 @@ def pre_accept_validate(self) -> None:
         """
         self.validate()
         states = exp_fetchers.get_exploration_by_id(self.target_id).states
-        if self.change.state_name not in states:
+        if self.change_cmd.state_name not in states:
             raise utils.ValidationError(
                 'Expected %s to be a valid state name' %
-                self.change.state_name)
+                self.change_cmd.state_name)
 
     def _get_change_list_for_accepting_edit_state_content_suggestion(
         self
     ) -> List[exp_domain.ExplorationChange]:
-        """Gets a complete change for the SuggestionEditStateContent.
+        """Gets a complete change_cmd for the SuggestionEditStateContent.
 
         Returns:
             list(ExplorationChange). The change_list corresponding to the
             suggestion.
         """
-        change = self.change
+        change = self.change_cmd
         exploration = exp_fetchers.get_exploration_by_id(self.target_id)
         old_content = (
-            exploration.states[self.change.state_name].content.to_dict())
+            exploration.states[self.change_cmd.state_name].content.to_dict())
 
         change.old_value = old_content
         change.new_value['content_id'] = old_content['content_id']
 
         return [change]
 
     def populate_old_value_of_change(self) -> None:
-        """Populates old value of the change."""
+        """Populates old value of the change_cmd."""
         exploration = exp_fetchers.get_exploration_by_id(self.target_id)
-        if self.change.state_name not in exploration.states:
+        if self.change_cmd.state_name not in exploration.states:
             # As the state doesn't exist now, we cannot find the content of the
             # state to populate the old_value field. So we set it as None.
             old_content = None
         else:
             old_content = (
-                exploration.states[self.change.state_name].content.to_dict())
+                exploration.states[self.change_cmd.state_name].content.to_dict()) # pylint: disable=line-too-long
```

Comment:
Do not add pylint-disables. I have already mentioned this previously.

EDIT: I notice more cases below. I'm going to stop reviewing here. Please only re-request a review once you have removed all the pylint-disables that you added.

Reply:
I am sorry for this, I think I mistakenly left some of the pylint disables. I will check out the all the changes I made once more and make the requested changes.
Thank you

Reply:
Done